### ISSUE TITLE
How to convert to integer

### ISSUE THREAD

**Issue by mayobserve** [pinam45/dynamic_bitset#9]
I didn't find an api that converts to long long, similar to the standard library's bitset

**Comment by pinam45** [comment]
I suppose you are referring to [std::bitset<N>::to_ulong](https://en.cppreference.com/w/cpp/utility/bitset/to_ulong) and [std::bitset<N>::to_ullong](https://en.cppreference.com/w/cpp/utility/bitset/to_ullong).

Yes, there is no such function at the moment for the [sul::dynamic_bitset](https://pinam45.github.io/dynamic_bitset/classsul_1_1dynamic__bitset.html), I think it would be a nice feature to have, the only downside is overflows, the functions would have to throw exceptions.

I will work on it when I have more free time (seems like my Github Actions are rusty and require some adjustments, and maintaining a good CI is a headache).
If you need the conversion right now, you can work something out with [data()](https://pinam45.github.io/dynamic_bitset/classsul_1_1dynamic__bitset.html#adfe46260b7caff45a27990b37b771a98) function, especially if the block type you use for the dynamic_bitset is already a `unsigned long long`:

```cpp
sul::dynamic_bitset<unsigned long long> d(12, 0b0100010110111);
unsigned long long val = *d.data();
assert(val == 0b0100010110111);
```

**Comment by pinam45** [comment]
I just released [v1.3.0](https://github.com/pinam45/dynamic_bitset/releases/tag/v1.3.0) which adds `to_ulong()` and `to_ullong()` interger conversion functions:
    - [constexpr unsigned long sul::dynamic_bitset<Block, Allocator>::to_ulong() const](https://pinam45.github.io/dynamic_bitset/classsul_1_1dynamic__bitset.html#a9d442a993dacd1aca16e6a33a12a0960)
    - [constexpr unsigned long long sul::dynamic_bitset<Block, Allocator>::to_ullong() const](https://pinam45.github.io/dynamic_bitset/classsul_1_1dynamic__bitset.html#aed3cb081490d4f269de7f3bfc574f8c8)

usage example:
```c++
// declare bitset with 12 bits from a value
sul::dynamic_bitset<uint32_t> a(12, 0b0100010110111);

// conversion to string with 0 and 1
std::cout << "String representation (0 and 1): " << a.to_string() << std::endl;

// conversion to unsigned long
std::cout << "Numeric value (unsigned long): " << a.to_ulong() << std::endl;

// conversion to unsigned long long
std::cout << "Numeric value (unsigned long long): " << a.to_ullong() << std::endl;
```

output:
```
String representation (0 and 1): 100010110111
Numeric value (unsigned long): 2231
Numeric value (unsigned long long): 2231
```
